src/main/scala/uk/gov/homeoffice/cirium/services/feed/Cirium.scala

```diff
@@ -174,7 +174,8 @@ object Cirium {
   def amendCiriumFlightStatus(status: CiriumFlightStatus): CiriumFlightStatus = {
     if ((status.arrivalAirportFsCode.toUpperCase == "LBA" ||
```

Comment:
We could reduce this to
`Set("CWL", "HUY", "INV", "LBA").contains(status.arrivalAirportFsCode.toUpperCase)`

Or even better, add the drt-lib dependency and check if it's a single terminal port rather than listing specific ports here. That would future proof us

Reply:
Using set at moment .  Need to get DRT lib working for cross scala version to import in cirium as it is using 2.13 version

Reply:
:+1: